dandi/support/bids/validator.py

```diff
@@ -545,12 +545,14 @@ def write_report(
 
 def _find_dataset_description(my_path):
     candidate = os.path.join(my_path, "dataset_description.json")
-    if my_path == "/":
+    # Windows support... otherwise we could do `if my_path == "/"`.
+    if my_path == "/" or "\\" and "/" not in my_path:
```

Comment:
This needs to be `if my_path == "/" or my_path == "\\" ...`.  I also suggest you write either `if (my_path == "/" or my_path == "\\") and "/" not in my_path` or `if my_path == "/" or (my_path == "\\" and "/" not in my_path)` to make it clear which one you mean.

Reply:
@jwodder thanks for the feedback, but better disregard whatever I'm doing until I remove draft status. I seem to have run into some windows compatibility issue bigger than just this line, and I wouldn't want to waste your time on my troubleshooting :)

Also, on the general comment, `if my_path == "/" or my_path == "\\" ...` — this won't work, as the top level on windows ends up being `C:`... but again, pls don't worry about this.